casbin/rbac/default_role_manager/role_manager.py

```diff
@@ -289,13 +292,20 @@ def _get_role_manager(self, *domain):
 
     def _affected_role_managers(self, *domain):
         domain_pattern = self._get_domain(*domain)
-        return [
-            self.rm_map[domain_str]
-            for domain_str in self.rm_map.keys()
-            if match_error_handler(
-                self.domain_matching_func, domain_str, domain_pattern
-            )
-        ]
+
+        if self.domain_matching_func != None:
+            return [
+                self.rm_map[domain_str]
+                for domain_str in self.rm_map.keys()
+                if match_error_handler(
+                    self.domain_matching_func, domain_str, domain_pattern
+                )
+            ]
+        else:
+            if domain_pattern in self.rm_map:
+                return [self.rm_map[domain_pattern]]
+            else:
+                return []
```

Comment:
Maybe can return like before `if expression`. `return [self.rm_map[domain_pattern] if domain_pattern in self.rm_map]`.

Reply:
done